### PR TITLE
Update the archetypes with app.yaml and update configs

### PR DESCRIPTION
**Pull request by msmshariq**
Update the archetypes to reflect the introduction of 'app.yaml' for apps, and removing 'config.yaml' from apps & components.

### CODE REVIEW

File: archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -44,15 +44,15 @@
...
+            <artifactId>org.wso2.carbon.uuf.common.foundation.ui</artifactId>
             <version>1.0.0-SNAPSHOT</version>
             <type>zip</type>
             <classifier>uuf-component</classifier>
         </dependency>
         <dependency>
-            <groupId>org.wso2.carbon.uuf.sample</groupId>
-            <artifactId>org.wso2.carbon.uuf.sample.theme.default</artifactId>
+            <groupId>org.wso2.carbon.uuf.common</groupId>
+            <artifactId>org.wso2.carbon.uuf.common.theme.default.theme</artifactId>
```
Comment (Kishanthan) [inline archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml]: why we have "theme" repeated in the artifact id?

Comment (msmshariq) [inline archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml]: @Kishanthan - It comes from the artifact id in uuf-common project - https://github.com/wso2/carbon-uuf-common/blob/master/themes/org.wso2.carbon.uuf.common.theme.default.theme/pom.xml

Comment (Kishanthan) [inline archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml]: @Shan1024 , the naming is wrong here. We need to change it

**Comment by msmshariq** [comment]
@Kishanthan - Updated the theme artifactId in 314e0cbf973b2e753e8491f4ff41e40a9df982f3, please review and merge.

File: archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (msmshariq) [inline archetypes/uuf-application/src/main/resources/archetype-resources/pom.xml]: @Kishanthan - Updated the theme artifactId in 314e0cbf973b2e753e8491f4ff41e40a9df982f3, please review and merge.

File: archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apis: [] # No APIs.
+
+bindings: [] # No bindings.
+
+config:
+    appName: ${componentName}
```
Comment (this) [inline archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml]: Lets change this to `pageSize: 10`

File: archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    # User will be redirected to following URI after logging-in successfully.
```
Comment (this) [inline archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml]: We don't need following config entries `loginRedirectUri`, `registerRedirectUri`, `logoutRedirectUri`

File: archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml (hunk 2 above)
Comment (msmshariq) [inline archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml]: @this - done.

File: archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml (hunk 3 above)
Comment (msmshariq) [inline archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml]: @this -  done, review and merge the changes.

**Pull request merged**